torch/_decomp/decompositions_for_jvp.py

```diff
@@ -179,7 +179,7 @@ def native_layer_norm_backward(
         if len(outer_dim_indices) > 0:
             d_bias: Optional[Tensor] = torch.sum(grad_out, outer_dim_indices, False)
         else:
-            d_bias = grad_out
+            d_bias = grad_out.view_as(grad_out)
     elif bias is not None:
```

Comment:
What is going on in line 26?
```
# TODO: what if jit decompositions exists, should we just use it?
#       or do we want to have an explicit white list like functorch had
#       using special JVP_DECOMP DynamicLayerFront kernel
```

Reply:
The mechanism we are using to register decompositions doesn't seem to be exclusively used for jvp. So open question here is whether `torch/csrc/jit/runtime/decomposition_registry.cpp` is being used for other things.

Reply:
Okay, that would make sense. I would put what you wrote in that code comment there so that future people reading the code who weren't around for the " special JVP_DECOMP DynamicLayerFront kernel" have better context

Reply:
Good idea, updated